### DISCUSSION TITLE
How to add downloaded voice models?

### DISCUSSION THREAD

**Discussion by NicoSteyl** [JarodMica/ai-voice-cloning#8] [Q&A] [answered]
I was unable to find much documentation on this, and it might be a simple question. After downloading a trained voice model from AI-Hub, how would I add that model to the WebUI? The downloaded model comes with 2 files, a .pth (usually found in the "weights" folder) file and .index file (usually found in the "logs" folder), are these compatible with your tts, and if so, where should be place these files?
Some guidance on how to get voices downloaded and specifically how to install them (pre-trained, ready to go stuff) would be great.

**Comment by Jonathan-v-S** [comment]
Are you still struggeling with this or have you figured it out? The pth file shoud be enought to get it to work, you should put it in this directory <Path_to_your_voice_cloning_directory>\ai-voice-cloning\voices\<Put name of model here> and put the pth. file in the folder you just made with the name you can recognize the model by. Then launch the webui and refresh voice list. Then you should be able to click the model you just put in there.

**Reply by uranjekm2** [reply]
when I do that and try to generate I get this error
Something went wrong
Unsupported audio format provided: .pth

**Reply by JarodMica** [reply, marked as answer]
If you are using RVC models from AI-Hub, it should be placed inside of ai-voice-cloning/models/rvc_models.  Both .index and .pth files are fine in here as it'll parse them.  You can then use them on inference by selecting the checkbox and then choosing the model you'd like to use.

It should be noted that tortoise is TTS and RVC is S2S (speech to speech) so you can't use them directly.  The pipeline works by generating a speech audio file in tortoise, then converting it with RVC in the GUI.  

Tortoise models are not widely distributed, so you will most likely not find these out on the web in high availability.

**Reply by AngryBearr** [reply]
@JarodMica can you please help.
I have downloaded pre trained models from [here ](https://huggingface.co/SerCe/tortoise-tts-ruslan)and [here](https://huggingface.co/prog1322/tortoise-Nastya). As I understand these are tts models. 
Where would I put them and try the inference using those models?
I have tried the `/my-drive/ai-voice-cloning/training/name-of-the-model/finetune/models/name-of-the-model.pth` but when I refresh the model list it won't pop-up. I have also added the same folder names to `voices ` folder but it didn't work either. Can you please help me solve this mystery.

And also, if there is a link to on how I can finetune based on these models (links above) that would be really great. Thanks in advance and thank you for all of your hard work! ❤️

**Reply by AngryBearr** [reply]
I have solved it myself and have successfully used those models. Now I'll try to fune-tune them, I'll see how it goes.

Again, can't express how great that you have done this repo and have make it work and all the effort you have put into it. Thank you very much for all your work ❤️
